### PR TITLE
using Kiali API directly

### PR DESCRIPTION
**Pull request by zackzhangkai**
**What type of PR is this?**

/kind api-change
/kind cleanup

**What this PR does / why we need it**:

Use kiali pod instead of wrapping kiali code to reduce the development work of KubeSphere Project.

**Which issue(s) this PR fixes**:

https://github.com/kubesphere/kubesphere/issues/3143

/assign @zryfish

### CODE REVIEW

File: pkg/kapis/servicemesh/metrics/v1alpha2/handler.go [hunk 1]
```diff
@@ -43,17 +86,104 @@ func getWorkloadMetrics(request *restful.Request, response *restful.Response) {
...
 
-	handlers.WorkloadMetrics(request, response)
+	if len(KialiQueryUrl) == 0 {
+		handlers.WorkloadMetrics(request, response)
+	} else {
+		url := fmt.Sprintf("%s/kiali/api/namespaces/%s/workloads/%s/metrics?%s", KialiQueryUrl, namespace, workload, request.Request.URL.RawQuery)
+		resp, err := http.Get(url)
+		klog.V(4).Infof("Proxy workload metrics request to %s", url)
+
+		if err != nil {
```
Comment (zryfish) [inline pkg/kapis/servicemesh/metrics/v1alpha2/handler.go:96]: Don't Repeat Yourself, there are lots of code are exactly the same.

Comment (zackzhangkai) [reply pkg/kapis/servicemesh/metrics/v1alpha2/handler.go]: fixed

File: pkg/utils/net/net.go [hunk 2]
```diff
@@ -45,3 +49,33 @@ func GetRequestIP(req *http.Request) string {
 
 	return address
 }
+
+func GetData(request *restful.Request, response *restful.Response, url string) {
```
Comment (zryfish) [inline pkg/utils/net/net.go:53]: no need to move function here, we probably not going to use it elsewhere, I think place it in `pkg/kapis/servicemesh/metrics/v1alpha2/handler.go` would be better

Comment (zackzhangkai) [reply pkg/utils/net/net.go]: done

**Pull request merged**